README.md

```diff
@@ -14,39 +14,53 @@ An overview of the architecture is shown below:
+| --http-headers                     | strings | N/A              | Http headers to be sent with warm up requests.                                                                  |
+| --http-requests                    | strings | N/A              | Http request to be sent. Request is in '<http-method>:<path>[:body]' format. E.g. post:/ping:{"key": "value"}   |
+| --probe-liveness-path              | strings | /alive           | Warm up sidecar liveness probe path                                                                             |
+| --probe-port                       | int     | /8000            | Warm up sidecar port for liveness and readiness probe                                                           |
+| --probe-readiness-path             | string  | /ready           | Warm up sidecar readiness probe path                                                                            |
+| --profile-cpu                      | string  | N/A              | Name of the file where to write CPU profile data                                                                |
+| --profile-memory                   | string  | N/A              | Name of the file where to write memory profile data                                                             |
+| --request-delay-milliseconds       | int     | 50               | Delay in milliseconds between requests                                                                          |
+| --target-grpc-host                 | string  | localhost        | gRPC host to warm up                                                                                            |
+| --target-grpc-port                 | int     | 50051            | gRPC port for warm up requests                                                                                  |
+| --target-http-host                 | string  | http://localhost | Http host to warm up                                                                                            |
+| --target-http-port                 | int     | 8080             | Http port for warm up requests                                                                                  |
+| --target-insecure                  | bool    | false            | Whether to skip TLS validation                                                                                  |
+| --target-readiness-path            | string  | /ready           | The path used for target readiness probe                                                                        |
+| --target-readiness-timeout-seconds | int     | -1               | Timeout for target readiness probe                                                                              |
+| --timeout-seconds                  | int     | 60               | Time after which warm up will stop making requests                                                              |
```

Comment:
I believe it makes more sense to call this `duration-seconds` if this reflects the time for which warmup will be running.